cfme/tests/configure/test_access_control.py

```diff
@@ -446,41 +501,89 @@ def test_rolename_required_error_validation():
 def test_rolename_duplicate_validation():
     role = new_role()
     role.create()
-    with error.expected("Name has already been taken"):
+    with pytest.raises(RBACOperationBlocked):
         role.create()
 
+    # Navigating away from this page will create an "Abandon Changes" alert
+    # Since group creation failed we need to reset the state of the page
+    view = navigate_to(role.appliance.server, 'Dashboard')
+    assert view.is_displayed, "Unable to change view to Dashboard after duplicate role creation"
 
 @pytest.mark.tier(3)
 def test_delete_default_roles():
-    flash_msg = version.pick({
-        '5.6': ("Role \"{}\": Error during delete: Cannot delete record "
-                "because of dependent entitlements"),
-        '5.5': ("Role \"{}\": Error during \'destroy\': Cannot delete record "
-                "because of dependent miq_groups")})
+    """Test that CFME prevents a user from deleting a default role
+    when selecting it from the Access Control EVM Role checklist
+
+    Steps:
+        * Login as Administrator user
+        * Navigate to Configuration -> Role
+        * Try editing the group EvmRole-approver
+    """
     role = Role(name='EvmRole-approver')
-    with error.expected(flash_msg.format(role.name)):
+    with pytest.raises(RBACOperationBlocked):
         role.delete()
 
 
 @pytest.mark.tier(3)
+def test_delete_default_roles_all_selection():
+    """Test that CFME prevents a user from deleting a default role
+    when selecting it from the Access Control EVM role checklist
+
+    Steps:
+        * Login as Administrator user
+        * Navigate to Configuration -> Role
+        * Try deleting the group EvmRole-approver
+    """
+    role = Role(name='EvmRole-approver')
+    with pytest.raises(RBACOperationBlocked):
+        role.delete(all_selection=True)
+
+
+@pytest.mark.tier(3)
 def test_edit_default_roles():
+    """Test that CFME prevents a user from editing a default role
+    when selecting it from the Access Control EVM Role checklist
+
+    Steps:
+        * Login as Administrator user
+        * Navigate to Configuration -> Role
+        * Try editing the group EvmRole-auditor
+    """
+    role = Role(name='EvmRole-auditor')
+    newrole_name = "{}-{}".format(role.name, fauxfactory.gen_alphanumeric())
+    role_updates = {
+        'name': newrole_name}
```

Comment:
No good reason. I think previous iterations of the test had multiple items in the dictionary